docs/dev/remote-coding-environments.md

```diff
@@ -0,0 +1,29 @@
+# Remote Coding Environments For Gaffer
+
+Gaffer is now configured for remote coding environemnts such as GitHub
+Codespaces and Gitpod. This addition allows for an easier and faster way to
+contribute to Gaffer with no manual setup or need to download dependices to a
+local machine.
+
+## GitHub Codespaces
+To use GitHub Codespaces simply open the gaffer repsoitory and click the "Code"
+button drop down. You'll see an option labeled "Open with codespaces" Click on
+it.
+
+This will launch Codespaces with all the correct configuration needed to
+contribute to the Gaffer repository.
+
+For more information on GitHub Codespaces click
+[here](https://github.com/features/codespaces).
+
+## Gitpod
+To use Gitpod you can simply prefix any GitHub URL with **`gitpod.io/#`** and
+follow the steps from there. You can also install the
+[extension](https://www.gitpod.io/docs/configure/user-settings/browser-extension)
+on your web browser. This adds a button to GitHub that does the prefixing for
+you.
+
+Once on Gitpod, your environment will be set up with all the correct
+configuration needed to contribute to the Gaffer repository.
+
+For more information on Gitpod click [here](https://www.gitpod.io/).
```

Comment:
Would be good if this linked to the docs directly and [link context was provided](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-refs.html).